db/schema.rb

```diff
@@ -11,11 +11,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190403010304) do
+ActiveRecord::Schema.define(version: 20190423235125) do
 
   create_table "recipes", force: :cascade do |t|
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
+    t.integer  "day"
+    t.integer  "slot"
+    t.string   "title"
+    t.integer  "calories"
+    t.integer  "readyminutes"
+    t.integer  "price"
+  end
+
+  create_table "recipes_users", id: false, force: :cascade do |t|
```

Comment:
So each entry of this table consists of user-recipe pairs

Reply:
Right